File: benchmark/results/python-multilang/run-004/b-baseline/output/metaengine_demo/notification/services/notification_service.py
"""NotificationService for the notification domain."""
from typing import Optional

from metaengine_demo.notification.aggregates.notification import Notification


class NotificationService:
    """Service exposing notification operations."""

    def create(self, input: Optional[Notification]) -> Notification:
        """Create a new Notification."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Notification]:
        """Find a Notification by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Notification]:
        """List Notifications up to limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete a Notification by id."""
        raise NotImplementedError("not implemented")

File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/notification/services/NotificationService.java
package com.metaengine.demo.notification.services;

import com.metaengine.demo.notification.aggregates.Notification;
import java.util.List;

/** Application service exposing notification operations. */
public class NotificationService {
    /** Create a notification from the provided partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a notification by id. Returns null when not found. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} notifications. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a notification by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
